atbash_cipher.cpp

```diff
@@ -2,12 +2,44 @@
 
 namespace atbash_cipher {
 
-std::string encode(const std::string& /*text*/) {
-    return "";
+static constexpr size_t GROUP_SIZE = 5;
+
+inline bool should_insert_space(const std::string& encoded) {
+    return (encoded.length() % (GROUP_SIZE + 1) == GROUP_SIZE);
+}
+
+inline char atbash_transform(char c) {
+    if (std::isalpha(c)) {
+        return 'z' - (std::tolower(c) - 'a');
+    }
+    return c;
```

Comment:
I would use early return here though the function is small. Early returns read well at the top of the statements as exceptions from the main part of the code.

```suggestion
    if (!std::isalpha(c))
        return c;
        
    return 'z' - (std::tolower(c) - 'a');
```

Reply:
Fixed.